.evergreen/uninstall_check_windows.cmd

```diff
@@ -7,11 +7,8 @@ echo > %INSTALL_DIR%\lib\canary.txt
 
 dir %INSTALL_DIR%\share\mongo-cxx-driver
 
-set PATH=%PATH%;"c:\Program Files (x86)\MSBuild\14.0\Bin"
-set PATH=%PATH%;"c:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin"
-
 pushd build
-MSBuild.exe /m generate_uninstall\uninstall.vcxproj
+%CMAKE% --build . --target uninstall
```

Comment:
[This Windows task](https://evergreen.mongodb.com/task/cxx_driver_windows_2k8_release_uninstall_check_windows_patch_7a06a355915f2ac105349e7925aed43d460df5b6_5ee191e2850e615ee6ea37d2_20_06_11_02_07_30) appears to expand $CMAKE to the empty string:
```
[2020/06/11 02:36:07.091] C:\data\mci\598ae302535af5d363ca095dfc713e06\mongo-cxx-driver\build>--build . --target uninstall
[2020/06/11 02:36:07.091] '--build' is not recognized as an internal or external command,
[2020/06/11 02:36:07.091] operable program or batch file.
```

I think this just needs a call to `. .evergreen/find_cmake.sh` above.

Reply:
Good catch.  However, `. .evergreen/find_cmake.sh` won't work in a batch file.  I've simply defined the variable based on the standard location of the CMake binary on Evergreen hosts.